xAPISGProcessor-SIMVA.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# xAPI-SG Processor SIMVA"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Processor of traces following the Experience API for Serious Games Profile (xAPI-SG)\n",
+    "\n",
+    "**xAPI-SG main reference:**\n",
+    "*Applying standards to systematize learning analytics in serious games.\n",
+    "Ángel Serrano-Laguna, Iván Martínez-Ortiz, Jason Haag, Damon Regan, Andy Johnson, Baltasar Fernández-Manjón\n",
+    "Computer Standards & Interfaces 50 (2017) 116–123, http://dx.doi.org/10.1016/j.csi.2016.09.014*\n",
+    "\n",
+    "Further info on GitHub wiki page: https://github.com/e-ucm/rage-analytics/wiki/xAPI-SG-Profile"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following code:\n",
+    "- expects as input a JSON file with a list of xAPI-SG statements, obtained from SIMVA\n",
+    "- analyzes the xAPI-SG statements and fills an adaptation of the default set of visualizations\n",
+    "https://github.com/e-ucm/rage-analytics/wiki/Default-visualizations-teacher"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SIMVA connection parameters \n",
+    "\n",
+    "### Defined in *ipyauth-keycloak-demo.env* file:\n",
+    "\n",
+    "- `base_url`: URL pointing to SIMVA's SSO service (default: https://sso.simva.e-ucm.es)\n",
+    "- `simva_ca_file`: if SIMVA is installed locally this should point to a file with the CA certificate used by simva (usually located at $SIMVA_HOME/docker-stacks/data/tls/ca/rootCA.pem), None otherwise\n",
+    "- `client_id`: SIMVA's client used by jupyter (e.g. jupyter)\n",
+    "- `redirect_uri`: jupyter URL (e.g. http://localhost:8888/callback/)\n",
+    "\n",
+    "### After running the following cell, click on \"Sign in\" to enter your SIMVA credentials"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import requests as rq\n",
+    "\n",
+    "from ipyauth import Auth, ParamsKeycloak\n",
```

Comment:
It is not documented that these changes are needed: https://github.com/e-ucm/ipyauth/tree/keycloak